src/SoftGlobalScope.jl

```diff
@@ -118,6 +133,25 @@ scoping rules for the global variables defined in `m`, returning the new express
 """
 softscope
 
+"""
+    @softscope(expr)
+
+Apply "soft" scoping rules to the argument of the macro. For example
+```jl
+julia> s = 0;
+
+julia> @softscope for i = 1:10
+           s += i
+       end
+
+julia> s
+55
+```
+"""
+macro softscope(ast)
+    esc(softscope(__module__, ast))
```

Comment:
Isn't `__module__` new in Julia 0.7 (JuliaLang/julia#22064)?   SoftScope.jl has to support 0.6, where I guess you would use `current_module()`.